js/authentication.md

```diff
@@ -1229,11 +1229,15 @@ Amplify.configure({
 Note: An ID token is only returned if openid scope is requested. The access token can be only used against Amazon Cognito User Pools if aws.cognito.signin.user.admin scope is requested. The phone, email, and profile scopes can only be requested if openid scope is also requested.
 {: .callout .callout--info}
 
-#### Launching the Hosted UI
+#### Using Hosted UI in Web Environment
```

Comment:
Can we call this "OAuth and Hosted UI in Web Environment"?

Reply:
Done. Also changed "Using Hosted UI in React Native " to "OAuth and Hosted UI in React Native"